CHANGELOG.md

```diff
@@ -1,246 +1,261 @@
-For more detailed information about the changes see the history of the [repository](https://github.com/votca/csg/commits/master).
+For more detailed information about the changes see the history of the
+[repository](https://github.com/votca/csg/commits/master).
 
 ## Version 1.6-dev
 
-## Version 1.5 _SuperVictor_ (released 31.01.19)
- * enable gitlab CI
+## Version 1.5 *SuperVictor* (released 31.01.19)
```

Comment:
Huh, no italic in headers?

Reply:
I'm still trying to figure out how to use the different auto formatters. It appears that no one autoformatter can do everything.